_templates/footer.html

```diff
@@ -9,6 +9,6 @@
   This site is made for Aalto Scientific Computing,
   but we <a href="https://github.com/AaltoSciComp/scicomp-docs">welcome
   contributions</a> that make it more useful to others, too.  Built
-  with <a href="https://www.sphinx-doc.org/">Sphinx</a>.
+  with <a href="https://www.sphinx-doc.org/">Sphinx</a>. This website is partially conformant with the <a href="https://www.w3.org/WAI/standards-guidelines/wcag/">Web Content Accessibility Guidelines (WCAG)</a> level AA. <a href="/accessibility/">Read more about the accessibility of scicomp.aalto.fi</a>.
```

Comment:
We can just leave the bit that says "Read about the accessibility of scicomp.aalto.fi" if it takes too much space in the footer.